### PR TITLE
v20.10 docs for cgroup v2 and rootless

### PR DESCRIPTION
**Pull request by AkihiroSuda**
### Proposed changes
Adds Docker v20.10 changes related to cgroup v2 and rootless.

* Docker now supports cgroup v2 (both rootful and rootless)
* Rootless mode graduated from experimental
* New storage driver: fuse-overlayfs

### Unreleased project version (optional)

v20.10

### CODE REVIEW

**Comment by AkihiroSuda** [comment]
@thaJeztah @cpuguy83

File: config/containers/runmetrics.md [hunk 1]
```diff
@@ -78,10 +125,16 @@ in `docker ps`, its long ID might be something like
 look it up with `docker inspect` or `docker ps --no-trunc`.
 
 Putting everything together to look at the memory metrics for a Docker
-container, take a look at `/sys/fs/cgroup/memory/docker/<longid>/`.
+container, take a look at `/sys/fs/cgroup/memory/docker/<longid>/` on cgroup
```
Comment (cpuguy83) [inline config/containers/runmetrics.md:128]: Is this true with the default systemd cgroups?

File: config/containers/runmetrics.md [hunk 2]
```diff
@@ -64,6 +76,41 @@ the hierarchy mountpoint. `/` means the process has not been assigned to a
...
+
+Docker supports cgroup v2 experimentally since Docker 20.03.
+Running Docker on cgroup v2 also requires the following conditions to be satisfied:
+* containerd: v1.4 or later
+* runc: v1.0.0-rc91 or later
+* Kernel: v4.15 or later (v5.2 or later is recommended)
+
+Note that the cgroup v2 mode behaves slightly different from the cgroup v1 mode:
+* The default cgroup driver (`dockerd --exec-opt native.cgroupdriver`) is "systemd" on v2, "cgroupfs" on v1.
+* The default cgroup namespace mode (`docker run --cgroupns`) is "private" on v2, "host" on v1.
```
Comment (cpuguy83) [inline config/containers/runmetrics.md:111]: This is not merged yet, right?

Comment (AkihiroSuda) [reply config/containers/runmetrics.md:111]: merged: https://github.com/moby/moby/pull/40174/commits/19baeaca267d5710907ac1b3c3972d44725fe8ad

An unmerged one is only for old API clients: https://github.com/moby/moby/pull/41072

File: config/containers/runmetrics.md (hunk 1 above)
Comment (AkihiroSuda) [reply config/containers/runmetrics.md:128]: updated

**Review by cpuguy83 — DISMISSED** [review]
LGTM

**Comment by AkihiroSuda** [comment]
Updated for v20.10. 

@cpuguy83 @tiborvass @thaJeztah PTAL

**Comment by AkihiroSuda** [comment]
Rebased.

@usha-mandya @tiborvass @thaJeztah Can we get this merged before v20.10 GA or after?

**Comment by AkihiroSuda** [comment]
rebased

**Review by tiborvass — APPROVED** [review]

**Review by thaJeztah — APPROVED** [review]
LGTM

**Pull request merged**